radiography/src/main/java/radiography/ScanScope.kt

```diff
@@ -9,8 +9,8 @@ package radiography
  *  - [ScanScopes.FocusedWindowScope]
  *  - [ScanScopes.singleViewScope]
  */
-fun interface ScanScope {
+public fun interface ScanScope {
```

Comment:
? for explicitness?

Reply:
Yes, strict API mode requires all visibilities and types to be explicit. This mode had not been enabled, because we were clobbering the option to turn it on (the thing this PR fixes).